static/js/compose.js

```diff
@@ -463,7 +463,6 @@ function check_unsubscribed_stream_for_send(stream_name, autosubscribe) {
 
 function validate_stream_message_mentions(stream_id) {
     const stream_count = stream_data.get_subscriber_count(stream_id) || 0;
-    wildcard_mention = util.find_wildcard_mentions(compose_state.message_content());
 
     // check if wildcard_mention has any mention and henceforth execute the warning message.
     if (wildcard_mention !== null && stream_count > exports.all_everyone_warn_threshold) {
```

Comment:
Won't this line crash after the change here?

Reply:
No, wildcard_mention is declared as a global variable at the start.